Orange/modelling/base.py

```diff
@@ -0,0 +1,81 @@
+from Orange.base import Learner, Model
+
+
+class FitterMeta(type):
+    """Ensure that each subclass of the `Fitter` class overrides the `__fits__`
+    attribute with a valid value."""
+    def __new__(mcs, name, bases, kwargs):
+        # Check that a fitter implementation defines a valid `__fits__`
+        if kwargs.get('name', False):
```

Comment:
I still don't think checking `name` attribute is adequate. https://github.com/biolab/orange3/pull/1771/commits/e6a14f0728f632371a213d69e8c4e8f59fc371e7 in https://github.com/biolab/orange3/pull/1771 obsoletes redundant short names on `Learner` types. Can you instead check that the class' name equals 'Fitter' or that none of the bases is named 'Fitter' or similar?

Also, `kwargs` is actually _a dict of class attributes_, so `attrs` or `namespace` or similar might be more appropriate.